Description: Static link against libbfd because the ABI is not stable
 From binutils-dev:
 Note that building Debian packages which depend on the shared libbfd is
 Not Allowed.  Debian packages must not link libbfd dynamically.
Author: Yaacov Akiba Slama <ya@slamail.org>
Origin: upstream, https://github.com/facebook/hhvm/commit/d7a95c828395a024c76a887a6e5285729055a43d
Bug: https://github.com/facebook/hhvm/issues/3262
Reviewed-By: Paul Tarjan <pt@fb.com>
Last-Update: 2014-10-16

--- hhvm-3.2.0+dfsg1.orig/CMake/HPHPFindLibs.cmake
+++ hhvm-3.2.0+dfsg1/CMake/HPHPFindLibs.cmake
@@ -340,7 +340,7 @@ if (LINUX OR APPLE)
   FIND_LIBRARY (RESOLV_LIB resolv)
 endif()
 
-FIND_LIBRARY (BFD_LIB bfd)
+FIND_LIBRARY (BFD_LIB libbfd.a)
 FIND_LIBRARY (LIBIBERTY_LIB iberty)
 
 if (NOT BFD_LIB)
